build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
     apply from: 'build-tools/repositories.gradle'
 
     ext {
-        es_version = '6.7.1'
+        es_version = '7.0.0'
```

Comment:
Change version for OD to 1.0.0 on line 44

Reply:
Done.